ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.18__arm64-nightly.yaml

```diff
@@ -69,6 +69,12 @@ releases:
       product: ocp
       stream: nightly
       version: "4.18"
+  source:
```

Comment:
openshift-e2e-test-qe does not reference openshift-extended-upgrade-pre, so we do not need the setting here.

Reply:
it has openshift-upgrade-qe-test-heterogeneous, so keep it.